ui/raidboss/common_replacement.ts

```diff
@@ -5,7 +5,7 @@ import { Lang, NonEnLang } from 'resources/languages';
 // It's awkward to refer to these string keys, so name them as replaceSync[keys.sealKey].
 export const syncKeys = {
   // Match Regexes, NetRegexes, and timeline constructions of seal log lines.
-  seal: '(?<=00:0839:|00.*0839.*\\\\\|)(.*) will be sealed off(?: in (?:[0-9]+ seconds)?)?',
+  seal: '(?<=00:0839:|00\\|0839\\|)(.*) will be sealed off(?: in (?:[0-9]+ seconds)?)?',
```

Comment:
Is this really a correct match for NetRegexes? Game logs are `00|timestamp|0839|blah will be sealed off|hash`.

Reply:
So it should be this instead?

```ts
  seal: '(?<=00:0839:|00\\|[^|]*\\|0839\\|)(.*) will be sealed off(?: in (?:[0-9]+ seconds)?)?',
```

Reply:
```diff
-seal: '(?<=00:0839:|00\\|[^|]*\\|0839\\|)(.*) will be sealed off(?: in (?:[0-9]+ seconds)?)?',
+seal: '(?<=00:0839:|00\\|[^|]*\\|0839\\|\\|)(.*) will be sealed off(?: in (?:[0-9]+ seconds)?)?',
```

I think with one extra bar? I thought find-translations would balk at this, but I can't seem to.  I should probably write a common replacement unit test that verifies that this works properly with replacements for both kinds of regexes.

Reply:
Looks good. I'll get this updated.

![image](https://user-images.githubusercontent.com/6119598/121279657-e8261880-c8a2-11eb-9dc9-dce80995a671.png)